sandpack-client/src/client.ts

```diff
@@ -298,7 +298,7 @@ export class SandpackClient {
       showLoadingScreen: this.options.showLoadingScreen ?? true,
       skipEval: this.options.skipEval || false,
       clearConsoleDisabled: !this.options.clearConsoleOnFirstCompile,
-      logLevel: this.options.logLevel,
+      logLevel: this.options.logLevel || SandpackLogLevel.Debug,
```

Comment:
I think this will overwrite the `None` log level, suggested fix:

```suggestion
      logLevel: this.options.logLevel ?? SandpackLogLevel.Debug,
```